client/src/assets/images/icons/index.js

```diff
@@ -1,425 +1,164 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export const CohortIcon = () => (
+const Icon = ({ children, ...props }) => (
   <svg
-    aria-hidden="true"
-    className="svg-inline--fa fa-users fa-w-20"
-    data-icon="users"
+    className="svg-inline--fa fa-w-20"
     data-prefix="fas"
     role="img"
```

Comment:
Consider using `aria-role` as well.

Reply:
aria-role is invalid attribute.
Eslint also warns about this: 
![Zrzut ekranu 2019-08-12 o 11 24 51](https://user-images.githubusercontent.com/27632432/62855946-d51c7e80-bcf3-11e9-8e31-af74f82ac068.png)

https://www.w3.org/WAI/PF/aria-1.1/states_and_properties

Reply:
Ok